src/api/app/views/webui/request/show.html.haml

```diff
@@ -2,6 +2,10 @@
   content_for(:meta_title, "Request #{@bs_request.number} (#{@bs_request.state})")
   content_for(:meta_description, @bs_request.description)
   content_for(:meta_image, gravatar_url(User.find_by_login(@bs_request.creator).email))
+  @pagetitle = "Request #{@bs_request.number}"
+  if @actions.count == 1
```

Comment:
Prefer to use a clause guard when it is possible:

```@pagetitle += ": #{@actions.first[:name]}" if @actions.count == 1```

I think rubocop doesnt lint it on haml, but it would be a cop violation on another area of our codebase